ouroboros-consensus/src/Ouroboros/Consensus/Node/Run/Mock.hs

```diff
@@ -44,6 +46,8 @@ instance ( ProtocolLedgerView (SimpleBlock SimpleMockCrypto ext)
       --  This doesn't matter much
       dummyDate = UTCTime (fromGregorian 2019 8 13) 0
 
+  nodeProtocolMagicId    = \_ _ -> ProtocolMagicId 0
```

Comment:
Should we provide a different value for the different protocols? We could introduce a type class for this that has an instance for the `SimpleBlock` instantiated to each protocol.